tests/unit/cli/test_main.py

```diff
@@ -216,7 +217,20 @@ def test_can_run_the_shell_action_for_a_specific_container(
             return_value=get_project(os.path.join(FIXTURE_ROOT, 'project01')))
         Nomad()
         assert mock_project_shell.call_count == 1
-        assert mock_project_shell.call_args == [{'container_name': ['c1', ], }, ]
+        assert mock_project_shell.call_args == [{'container_name': ['c1', ], 'username': None}, ]
+
+    @unittest.mock.patch(
```

Comment:
for the record: I'm not yes familiar with the whole test mechanism, but I find this boilerplate a bit heavy (lots of repetition). I'll look into parametrizing the thing when I have the chance.